### PR TITLE
#556  Init root categories removing

### LINKED ISSUE #556: Rm high level categories
37 subcategories of the second level will become the high-level categories.

- [ ] Create more root categories with test_db to test it's presense at the UI.
- [ ] Move list to a matrix at the dropdown http://192.168.56.10:8020/gbi/
- [ ] Move list to a matrix on the catalog page http://192.168.56.10:8020/gbi/
- [ ] Deploy changes
- [ ] Create and run a script to move categories on the prod

### PR DESCRIPTION
**Pull request by duker33**
Closes #556

### CODE REVIEW

File: stroyprombeton/tests/tests_views.py [hunk 1]
```diff
@@ -267,7 +267,7 @@ def test_fetch_positions_searching(self):
 
     def test_products_are_from_category(self):
```
Comment (artemiy312) [inline stroyprombeton/tests/tests_views.py]: Let's create todo to fix these buggy tests. They should rely upon hard-coded data

File: stroyprombeton/views/catalog.py [hunk 2]
```diff
@@ -70,8 +70,12 @@ def serialize_categories(categories):
     return response
 
 
+# @todo #556:120m  Show root categories as matrix.
```
Comment (artemiy312) [inline stroyprombeton/views/catalog.py]: I believe 60m will be enough, but up to you

Comment (duker33) [reply stroyprombeton/views/catalog.py]: @artemiy312 , if we have mockup, it's true. But here some UX thoughts would be realized

File: stroyprombeton/tests/tests_views.py (hunk 1 above)
Comment (artemiy312) [reply stroyprombeton/tests/tests_views.py]: shouldn't *rely

**Pull request merged**